src/Microsoft.TemplateEngine.Abstractions/Constraints/TemplateConstraintInfo.cs

```diff
@@ -6,12 +6,12 @@
 namespace Microsoft.TemplateEngine.Abstractions.Constraints
 {
     /// <summary>
-    /// Represents information about template constraint defintion in template cache <see cref="ITemplateInfo"/>.
+    /// Represents information about template constraint definition in template cache <see cref="ITemplateInfo"/>.
     /// </summary>
     public class TemplateConstraintInfo
     {
         /// <summary>
-        /// Creates instance of the clase.
+        /// Creates instance of the class.
```

Comment:
Should this perhaps be `Creates a new <see cref="TemplateConstraintInfo"/> instance.`